packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +84,27 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+/// An expansion panel that allows for radio-like functionality.
+///
+/// A unique identifier [value] must be assigned to each panel.
+class ExpansionPanelRadio extends ExpansionPanel {
+
+  /// A subclass of expansion panel that allows for radio functionality.
```

Comment:
Just say "An expansion panel" like you had above.  Don't put the "subclass of" part in because if that changes, it's easy for the docs to get missed and be out of date.